ReleaseTooling/Sources/ZipBuilder/CarthageUtils.swift

```diff
@@ -286,12 +291,14 @@ extension CarthageUtils {
     }
 
     // Write the Info.plist.
-    generatePlistContents(forName: "Firebase", to: frameworkDir)
+    generatePlistContents(forName: "Firebase", withVersion: version, to: frameworkDir)
   }
 
-  static func generatePlistContents(forName name: String, to location: URL) {
+  static func generatePlistContents(forName name: String,
+                                    withVersion version: String,
+                                    to location: URL) {
     let plist: [String: String] = ["CFBundleIdentifier": "com.firebase.Firebase-\(name)",
-                                   "CFBundleInfoDictionaryVersion": "6.0",
+                                   "CFBundleInfoDictionaryVersion": version,
```

Comment:
I'm not sure this should be the pod version, this is the version of the Info.plist structure: https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleinfodictionaryversion?language=objc

Instead I think we should change the [`CFBundleVersion`](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion?language=objc) and maybe even [`CFBundleShortVersionString`](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleshortversionstring?language=objc)

Reply:
Good catch! The 6.0 threw me off :)